com.bixense.PasswordCalculator.json

```diff
@@ -0,0 +1,60 @@
+{
+	"app-id": "com.bixense.PasswordCalculator",
+	"runtime": "org.gnome.Platform",
+	"runtime-version": "3.22",
+	"sdk": "org.gnome.Sdk",
+	"command": "pwcalculator",
+	"finish-args": [
+		"--socket=x11",
+		"--socket=wayland"
+	],
+	"cleanup": [ "/include", "/bin/wx*" ],
```

Comment:
It installs a lot more unused files than this, so look through `builddir/files` and remove the rest.

Reply:
I only found the folder `/lib/wx`. Which files do you have in mind?

Reply:
Also `/share/aclocal`